source/catalog.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="icon" href="favicon.ico">
+  <link rel="icon" href="img/favicons/icon.svg" type="image/svg+xml">
+  <link rel="apple-touch-icon" href="img/favicons/apple.png">
+  <link rel="manifest" href="manifest.webmanifest">
+  <title>Магазин игрушек «Мишка»</title>
+</head>
+<body class="page__body">
+
+  <header class="page__page-header page-header">
+    <div class="page-header__container container">
+      <nav class="page-header__navigation navigation">
+        <a class="navigation__link-logo logo" href="index.html">
+          <img class="logo__icon" src="img/logo-mobile.svg" width="86" height="35" alt="">
+        </a>
+        <button class="navigation__toggle">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <ul class="navigation__list site-list">
+          <li class="site-list__item">
+            <a class="site-list__link" href="#">Каталог товаров</a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link" href="#">Вязание на заказ</a>
+          </li>
+        </ul>
+        <ul class="navigation__list user-list">
+          <li class="user-list__item user-list__item--search">
+            <a class="user-list__link" href="#">Поиск по сайту</a>
+          </li>
+          <li class="user-list__item user-list__item--basket">
+            <a class="user-list__link" href="#">Корзина: пока пуста</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main class="page__page-main page-main">
+    <section class="page-main__catalog catalog">
+      <div class="catalog__container container">
+        <h1 class="catalog__title">Каталог товаров</h1>
+      </div>
```

Comment:
Контейнер не обязательно всегда миксовать с элементом контейнера (собственно, так теряется его смысл как отдельного блока).
Можно иногда и с другим элементом:
```suggestion
      <h1 class="catalog__title container">Каталог товаров</h1>
```